discussion_agents/cog/functional/react.py

```diff
@@ -20,19 +17,21 @@ def _build_agent_prompt(question: str, scratchpad: str) -> str:
     Args:
         question (str): The question to be included in the prompt.
         scratchpad (str): Additional scratchpad information to be included.
-
+        examples (str): The example as a guide of how the test should be prompted.
+        instruction (str) : The template of the prompt.
```

Comment:
```suggestion
        instruction (str): The template of the prompt.
```

Also, be more specific in docstrings.

Reply:
Probably should rename instruction to `prompt_template` or something more representative of what it's purpose is.